build/book-review-config-develop.json

```diff
@@ -4,9 +4,9 @@
   "id": "01FP2SME0ENTXWF362XHM6Z1B4",
```

Comment:
It would be great if you can leave out all the build/* files that are not related to vimeo.
Then it will not trigger a reload of the other templates in running clients.

Unfortunately, the webpack script I wrote is not very helpful with this.
So I normally avoid committing the build files that are not related to the changes I have made :)
I will look into a better approach for this.

Reply:
Thank you for explaining how you normally do. If you don't have time to work on the Webpack script, we can just add your explanation to the template creation documentation. That will be a perfectly fine solution.